### PR TITLE
[t115423] Improving mail creation

### PR DESCRIPTION
**Pull request by BT-aestebanez**
mail_operating_unit

 
<div> Links to Odoo: </div> <ul>
<li><a target="_blank" href="https://odoo.braintec-group.com/web#view_type=form&model=project.task&id=115423">[t115423] [MIUS] [DEV] [mt1935] Biochemistry USA - Follow up report</a></li>
</ul>
<div>Affected Modules:</div>
<table><thead><tr><th>Module</th><th>Ext</th></tr></thead>
<tr><td>mail_operating_unit</td><td>.py</td></tr>
</tbody></table>

### CODE REVIEW

**Review by BT-rmartin — CHANGES_REQUESTED** [review]
Please, check that there are problems with OCA travis about length of lines. 
Coverage is also decreasing so consider extending or adding tests.

File: mail_operating_unit/models/mail_mail_ext.py [hunk 1]
```diff
@@ -24,18 +24,33 @@ def create(self, vals_list):
...
+                        self.env.context.get('uid')
+                    )
+                else:
+                    user = self.env.user
                 operating_unit = user.operating_unit_for_mails_id
 
-            if operating_unit and operating_unit.catchall_alias and operating_unit.catchall_domain:
-                sender_email_address = (operating_unit.catchall_alias + '@' +
-                                        operating_unit.catchall_domain)
+            if operating_unit and operating_unit.catchall_alias and \
```
Comment (BT-jmontero) [inline mail_operating_unit/models/mail_mail_ext.py]: Use parenthesis instead of the back slash in order to split IFs in multiple lines.

**Review by BT-jmontero — APPROVED** [review]
In general, it looks good. The only thing I see is that you are restricting the length of the lines below BT's limit (115), which causes the file to be longer than needed.

**Comment by BT-aestebanez** [comment]
That's why this repo is OCA's and it has the limit on 79 chars

File: mail_operating_unit/models/mail_mail_ext.py (hunk 1 above)
Comment (BT-aestebanez) [reply mail_operating_unit/models/mail_mail_ext.py]: Done

**Pull request merged**